scripts/qemu/ia32-generic.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+#
+# Shell script for running Phoenix-RTOS on QEMU (ia32-generic)
+#
+# Copyright 2021 Phoenix Systems
+# Author: Lukasz Kosinski
+#
+
+CMD="qemu-system-i386"
+
+if [ "$#" -eq 0 ]; then
+	# No args (default command)
+	CMD+=" -hda ../../_boot/phoenix-ia32-generic.disk"
+else
+	hd=0
+	vblk=0
+	# Parse args
+	while [ "$#" -gt 0 ]; do
+		case "$1" in
+			# Load kernel (.elf)
+			-kernel)
+				CMD+=" -kernel $2"
+				shift 2
+				;;
+			# Redirect serial port
+			-serial)
+				CMD+=" -serial $2"
+				shift 2
+				;;
+			# Add ATA drive
+			-ata|-ide)
+				CMD+=" -drive file=$2,if=none,id=hd$hd -device ide-hd,drive=hd$((hd++))"
+				shift 2
+				;;
+			# Add VirtIO drive (modern)
+			-vblk|-virtio-blk)
+				CMD+=" -drive file=$2,cache=unsafe,if=none,id=vblk$vblk -device virtio-blk-pci,disable-legacy=on,disable-modern=off,drive=vblk$((vblk++))"
+				shift 2
+				;;
+			# Add VirtIO GPU
+			-vgpu|-virtio-gpu)
+				CMD+=" -device virtio-gpu-pci"
+				shift
+				;;
+			*)
+				echo "$1: unknown option"
```

Comment:
no help or example? so I need to read the whole script to understand how to use it?

I see You've attached the sample usage to the PR description but that would be lost when PR would be merged.

Maybe we should simultaneously add the necessary description/examples in [phoenix-rtos-doc](https://github.com/phoenix-rtos/phoenix-rtos-doc/tree/master/quickstart) repo?

Reply:
I've added help option to the main scripts (-help or -h). I agree that we should add documentation to phoenix-rtos-doc. I'll do that in another PR and mention this one.